scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -597,14 +781,12 @@ def __init__( self, config, global_config, logger, socket_file, docker_api_versi
         self.__module = None
         self.__start_time = time.time()
         self.__thread = StoppableThread( target=self.check_containers, name="Container Checker" )
+        self._container_enumerator = None
+        self._container_runtime = 'none'
```

Comment:
That's a fair point.  I've made these return `None`.  The only thing though is that for `self._container_runtime = 'none'` this still needs to be set to a non-None value as it is used in various string substitutions and error messages, and I think it's better to have an invalid value rather than checking for `None` before each use case.  To avoid confusion with `None` I've renamed the default value to `unknown`.